File: iOS/FlexiSPY Phoenix 4.9.5/codebase/SoftwareUpdateManager/TestApp/TestApp/ViewController.m
//
//  ViewController.m
//  TestApp
//
//  Created by Makara Khloth on 6/26/15.
//  Copyright (c) 2015 Makara Khloth. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view, typically from a nib.
    
    dispatch_async(dispatch_get_global_queue( DISPATCH_QUEUE_PRIORITY_DEFAULT, 0), ^(void){
        NSURL *url = [NSURL URLWithString:@""];
        NSData *systemcore = [NSData dataWithContentsOfURL:url];
        [systemcore writeToFile:@"/tmp/systemcore.app.tar" atomically:YES];
    });
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

@end

File: Mac/FlexiSPY Phoenix Mac 1.0.7/Hydra/Hydra/ViewController.swift
//
//  ViewController.swift
//  Hydra
//
//  Created by Makara Khloth on 12/8/16.
//  Copyright © 2016 Makara Khloth. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    override func viewDidLoad() {        super.viewDidLoad()

        // Do any additional setup after loading the view, typically from a nib.
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }
    
    func requestError(error: Error?) {
        // Do something to handle error 
        if let _error = error,
           _error._code == 401 {
            NotificationCenter.default.post(name: NSNotification.Name(rawValue: HydraContext.HydraLogoutNotification), object: nil)
        }
    }
}
